ts/src/test/static/data/p2b.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
@samgermain we should always add static tests when adding a new exchange/method/etc